packages/ui/src/components/Visualization/Custom/CustomNode.tsx

```diff
@@ -19,25 +19,26 @@ import { ItemReplaceNode } from './ItemReplaceNode';
 interface CustomNodeProps extends WithSelectionProps {
   element: Node<CanvasNode, CanvasNode['data']>;
 }
+const noopFn = () => {};
 
 const CustomNode: FunctionComponent<CustomNodeProps> = ({ element, ...rest }) => {
   const vizNode = element.getData()?.vizNode;
   const label = vizNode?.getNodeLabel();
-  const [statusDecoratorTooltip, setStatusDecoratorTooltip] = useState(vizNode?.getNodeStatusMessage());
-  vizNode?.addNodeStatusListener((status: NodeStatus, message: string | undefined) => {
-    element.setNodeStatus(status);
-    setStatusDecoratorTooltip(message);
-  });
+  const statusDecoratorTooltip = vizNode?.getNodeValidationText();
+  const nodeStatus = !statusDecoratorTooltip ? NodeStatus.default : NodeStatus.warning;
+
+  console.log(`CustomNode ${vizNode?.id} - Rendering`);
```

Comment:
Shouldn't we remove the console.log?

Reply:
Yes, that would be very ideal, yes :face_exhaling: , my bad, I'll remove it right away

Reply:
Done :+1: